### PR TITLE
Add .eslintrc.js files to blueprints for apps and addons

### LINKED ISSUE #6864: Include ESLint default config files in app/addon blueprints
Instead of running the `ember-cli-eslint` blueprint on `ember new` we should instead just duplicate the empty ESLint config files so that e.g. they show up at https://github.com/ember-cli/ember-new-output and also work with the `--skip-npm` flag.

### PR DESCRIPTION
**Pull request by rwwagner90**
I copied over the .eslintrc.js files to the relevant places, to the best of my knowledge. Please let me know if I need to make any adjustments.

Resolves #6864

### CODE REVIEW

File: blueprints/addon/files/.eslintrc.js [hunk 1]
```diff
@@ -0,0 +1,13 @@
+module.exports = {
```
Comment (rwjblue) [inline blueprints/addon/files/.eslintrc.js]: This file should be inherited from the app blueprint. I don't think we need it here.

**Comment by rwjblue** [comment]
Also, can you retarget this PR at beta branch (it should've possible via the github web UI with out closing and reopening now)?

**Comment by rwwagner90** [comment]
@rwjblue I changed the target, but now I have all the other changes from master in here it appears. Should I just close, branch off of beta and reopen?

**Comment by rwjblue** [comment]
Hmm. Yes, sorry.

**Pull request closed without merging**